cfpb/capital-framework
# Bugfix for GitHub flavored Markdown

GitHub style heading links weren't working. Turned out we were converting the files to html outside of kramdown. Updated Gulp to only copy the Markdown files and the Kramdown config to use the GitHub style settings.

## Removals

- Removed the Gulp usage task and related packages

## Changes

- Added usage docs to the Gulp copy task
- Updated Jekyll config to use GitHub settings in Kramdown

## Testing

- If you've never run the gh-pages branch, run `npm install`
- Run `npm run build && npm start`
- Navigate to the components and check that they display properly, including GitHub style code blocks

## Review

- @ascott1 
- @Scotchester 

## Checklist
